inlong-sort/sort-formats/format-common/src/main/java/org/apache/inlong/sort/formats/inlongmsg/IgnoreFailureHandler.java

```diff
@@ -27,6 +27,11 @@ public class IgnoreFailureHandler implements FailureHandler {
 
     private static final Logger LOG = LoggerFactory.getLogger(IgnoreFailureHandler.class);
 
+    @Override
+    public void onParsingMsgFailure(Object msg, Exception exception) {
+        LOG.error("Could not properly serialize msg=[{}].", msg, exception);
```

Comment:
I believe parse is equal to deserialize rather than serialize

Reply:
done